### ISSUE TITLE
Order TODOs manually (for prioritisation)

### ISSUE THREAD

**Issue by willnwhite** [mrodalgaard/atom-todo-show#127]
Each time I look at my list of TODOs I have to read through them all to pick out the next one to do. I'd like to order them as I go, so that once I've done one, I can open up the list and see the next one to do.

**Comment by mrodalgaard** [comment]
You could use the tag or id fields like below, add it to the "Show In Table" config and sort after that.

```
// TODO(1): This todo will have id 1
// TODO(2): This todo will have id 2
// TODO: This todo will have tag 1 #1
// TODO: This todo will have tag 2 #2
```

**Comment by willnwhite** [comment]
Thanks, I will try this out. I guess the most intuitive and least work is to be able to drag and drop TODOs in the list, especially when it comes to re-ordering.

**Comment by onetrev** [comment]
Agreed that Drag and Drop ordering would be great. I imagine adding that at this stage could be a lot of work though. How the IMDONE package enables it is nice for sure.

**Comment by dmig** [comment]
Small proposal: make enhancement to sort by type, get sort order from FindTheseTodos field
